lib/Geo/ShapeContainer.cpp

```diff
@@ -55,6 +55,24 @@
 using namespace arangodb;
 using namespace arangodb::geo;
 
+namespace {
+
+S2Polygon latLngRectToPolygon(S2LatLngRect const* rect) {
+  // Construct polygon from rect:
+  std::vector<S2Point> v;
+  v.reserve(5);
+  v.emplace_back(Coordinate::fromLatLng(rect->GetVertex(0)).toPoint());
+  v.emplace_back(Coordinate::fromLatLng(rect->GetVertex(1)).toPoint());
+  v.emplace_back(Coordinate::fromLatLng(rect->GetVertex(2)).toPoint());
+  v.emplace_back(Coordinate::fromLatLng(rect->GetVertex(3)).toPoint());
+  v.emplace_back(Coordinate::fromLatLng(rect->GetVertex(0)).toPoint());
```

Comment:
```suggestion
  v.emplace_back(rect->GetVertex(0).toPoint());
  v.emplace_back(rect->GetVertex(1).toPoint());
  v.emplace_back(rect->GetVertex(2).toPoint());
  v.emplace_back(rect->GetVertex(3).toPoint());
  v.emplace_back(rect->GetVertex(0).toPoint());
```

Reply:
Actually, it has to be `ToPoint` with a capital T, but then it works. The `S2LatLng` object only has a `ToPoint` method, which I missed. Great!